### PR TITLE
Home page tabs

### PR DESCRIPTION
**Pull request by leonplondon**
# Summary
Added tabs to home screen

# Trello Card
https://trello.com/c/vudkYfrY/15-u4-home-page-tabs-de-navegaci%C3%B3n

# Screenshots
screenshot_1543444813

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/home/news/INewsView.kt
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/news/INewsView.kt:4]: Please don't leave interfaces or classes empty. Will be completed in another branch?

File: app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt:7]: Same question as INewsView

File: app/src/main/java/ar/com/wolox/android/example/ui/home/profile/IProfileView.kt
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/profile/IProfileView.kt:4]: Same as question above

File: app/src/main/java/ar/com/wolox/android/example/ui/home/profile/ProfilePresenter.kt
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/profile/ProfilePresenter.kt:7]: Same as questions above

File: app/src/main/res/layout/activity_home.xml
Comment (lmiotti) [inline app/src/main/res/layout/activity_home.xml]: Why are you using a Toolbar here? We are only wanting to add and icon and a text

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt
Comment (lmiotti) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt]: Move this attribute below @Inject.

**Review by lmiotti — CHANGES_REQUESTED** [review]
Some minor changes. You're doing it great. Keep on like this

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt
Comment (dylan-muszel) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt]: Here you can omit the type of the declaration. Kotlin supports type inference which means that the compiler find out types from context.
It would look like this: `override fun layout() = R.layout.activity_home` and the Android Studio will put the `: Int` by itself.

File: app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt
Comment (dylan-muszel) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt]: Same as above.

File: app/src/main/java/ar/com/wolox/android/example/ui/home/profile/ProfileFragment.kt
Comment (dylan-muszel) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/profile/ProfileFragment.kt]: Same as above.

**Review by dylan-muszel — COMMENTED** [review]
It's great! There's only one thing about Kotlin that I commented.

File: app/src/main/java/ar/com/wolox/android/example/ui/home/news/INewsView.kt
Comment (leonplondon) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/news/INewsView.kt:4]: Added TODO comment

File: app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt
Comment (leonplondon) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt:7]: Added TODO comment

File: app/src/main/java/ar/com/wolox/android/example/ui/home/profile/IProfileView.kt
Comment (leonplondon) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/profile/IProfileView.kt:4]: Added TODO comment

File: app/src/main/java/ar/com/wolox/android/example/ui/home/profile/ProfilePresenter.kt
Comment (leonplondon) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/profile/ProfilePresenter.kt:7]: Added TODO comment

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt
Comment (leonplondon) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt]: Reorganized elements, first injected fields then the others.

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt
Comment (leonplondon) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt]: Removed type declaration

File: app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt
Comment (leonplondon) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt]: Removed type declaration

File: app/src/main/java/ar/com/wolox/android/example/ui/home/profile/ProfileFragment.kt
Comment (leonplondon) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/profile/ProfileFragment.kt]: Fixed, removed type declaration

**Review by lmiotti — APPROVED** [review]

**Comment by lmiotti** [comment]
Great work! Before merging this PR wait until "Unit tests for login presenter" is approved and merged

**Pull request merged**
